packages/evm/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import { EOFContainer, validateEOF } from './eof/container.ts'
+import { EVMError, EVMErrorMessages as EVMErrorMessage } from './errors.ts'
```

Comment:
Why are we aliasing `EVMErrorMessages as EVMErrorMessage ` and then `EVMError as EvmError`? Is it because we don't want to update how the exports are named?

Reply:
That was an unnecessary alias, good catch! Fixed.